qa/tasks/cephfs/test_fsnew.py

```diff
@@ -0,0 +1,64 @@
+import errno
+import time
+from unittest import SkipTest
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from teuthology.orchestra.run import CommandFailedError
+
+class TestFSNew(CephFSTestCase):
```

Comment:
Could you move this file to test_misc.py please?  That way it will get picked up and run and we avoid the need to create a separate yaml as @tchaikov mentions (it makes our tests take a bit less time if we bundle things like that, it's all a bit arbitrary though!)